src/components/FocusZone/FocusZone.Props.ts

```diff
@@ -68,7 +68,7 @@ export interface IFocusZoneProps extends React.Props<FocusZone> {
 
   /**
    * Props mixed into the div root element that will be mixed into the root element, *before* other props are applied.
-   * This allows you to extend the root element with additional attributes, such as data-automation-id needed for
+   * This allows you to extend the root element with additional attributes, such as data-automationid needed for
```

Comment:
BTW it should be "data-automation-id" which is a pretty standard thing outside of MSFT. It just happens that a dev in a hurry checked in a bunch of data-automationid elsewhere because that's what their project used.

If you google "data-aut" you'll see autocomplete backing me up :)